theseus_gui/src/pages/Browse.vue

```
@@ -151,6 +151,80 @@ if (route.query.ai) {
+          val + `/members`,
+          'direct url members search results',
+          true
+        )
+        if (rawResults) {
+          if (rawResults.project_type != projectType.value) {
+            results.value = searchResults
+            return
+          }
+
+          // Convert to ProjectResult
+          searchResults.total_hits = 1
+          searchResults.hits = [
+            {
+              slug: rawResults.slug,
+              title: rawResults.title,
+              description: rawResults.description,
+              categories: rawResults.categories.concat(rawResults.additional_categories),
+              client_side: rawResults.client_side,
+              server_side: rawResults.server_side,
+              project_type: rawResults.project_type,
+              downloads: rawResults.downloads,
+              icon_url: rawResults.icon_url,
+              color: rawResults.color,
+              thread_id: rawResults.thread_id,
+              monetization_status: rawResults.monetization_status,
+              project_id: rawResults.id,
+              author: memberRawResults
+                .filter((mem) => mem.role === 'Owner')
+                .map((mem) => mem.user.username)[0],
+              display_categories: rawResults.categories,
+              versions: rawResults.game_versions,
+              follows: rawResults.followers,
+              date_created: new Date().toISOString(),
+              date_modified: rawResults.updated,
+              latest_version: rawResults.game_versions[rawResults.game_versions.length - 1],
+              license: rawResults.license.id,
+              gallery: rawResults.gallery.map((img) => img.url),
+              featured_gallery: rawResults.gallery
+                .filter((img) => img.featured === true)
+                .map((img) => img.url)[0],
+            },
+          ]
+        }
+      }
+    }
+    results.value = searchResults
+
+    // short circuit
+    return
+  } catch (e) {
+    // ignore
```

ignore on error?

---

This code was intended to be a quick and simple implementation of URL searching. If you take into consideration the context of the code, and follow the logic as to what occurs when there's an error, the only error that would realistically occur here is an invalid URL. The user is already receiving this feedback with no search results appearing as a result. Thus, the error is ignored because there is already user feedback.